engines/native/sandbox.go

```diff
@@ -35,6 +38,13 @@ func newSandbox(b *sandboxBuilder) (*sandbox, error) {
 		return nil, fmt.Errorf("Failed to temporary folder, error: %s", err)
 	}
 
+	if b.payload.Context != "" {
+		if err = fetchContext(b.payload.Context, homeFolder.Path()); err != nil {
```

Comment:
In a world where we cache a reuse downloaded resources, we wouldn't download to the user folder.
Instead we would download to some temporary folder and extract from there every time we need it.

Like said, I think this is just TODOs as the subsystems for this isn't in place yet.